.github/workflows/basic.yml

```diff
@@ -68,10 +74,31 @@ jobs:
           python-version: '3.10'
       - uses: pre-commit/action@v3.0.1
         timeout-minutes: 3
+
+  base_image_build:
```

Comment:
what about to call this step inside general dockerize step  ./.github/actions/dockerize-neon-tests ?

because except this pipeline with basic/evm tests we have several where we use uses: ./.github/actions/dockerize-neon-tests as well (operator tests, OZ, dapps) and you should change them as well